project/README.rst

```diff
@@ -93,3 +93,6 @@ Project List by Account Name
 
 #. `F84064014 <https://github.com/F84064014>`__ for
    `SimpleNN <README.rst>`__: https://github.com/F84064014/SimpleNN
+
+#. `BlenderWang9487 <https://github.com/BlenderWang9487>`__ for
+   `DummyML <README.rst>`__: https://github.com/BlenderWang9487/DummyML
```

Comment:
Could you point the `README.rst` to your proposal file, i.e., `BlenderWang9487/README.rst`?  Sorry I wasn't clear in the very beginning and the example wasn't clear enough either.

And please help resolve the merge conflict.

Reply:
Sure! But I might revert some commits because I think I messed up some of it by accidentally merging the `proposal` branch to my `BlenderWang9487-proposal-submission` branch. Sorry about that!

Reply:
Thanks a lot.  And no problem.  After I merge you can make amendments any time.  You are encouraged to continuously change the proposal.

Reply:
My branch is messed up more, but I think there is no conflict now.

I think I will delete it and create a new one after merging. Thanks!

Reply:
This proposal is meant for you to practice the git tricks.  At this point, it's good enough that you managed to have the doc ("code") merged.  Just keep improving your git-fu.